src/Utilities/Utils.res

```diff
@@ -582,10 +584,12 @@ let constructClass = (~classname, ~dict) => {
 let generateStyleSheet = (classname, dict, id) => {
   let createStyle = () => {
     let style = document["createElement"]("style")
-    style["type"] = "text/css"
+    let style1 = Window.createElement("style")
+    // style["type"] = "text/css"
     style["id"] = id
     style["appendChild"](document["createTextNode"](constructClass(~classname, ~dict)))->ignore
-    document["body"]["appendChild"](style)->ignore
+    let obj = document["body"]
+    obj["appendChild"](style)->ignore
```

Comment:
Can you explain changes done in this file? Seems redundant

Reply:
update this